src/cogs/music_commands.py

```diff
@@ -9,26 +9,24 @@
 from spotipy.oauth2 import SpotifyClientCredentials
 
 from internal.lavalink_voice_client import LavalinkVoiceClient
-from internal.bot import BotBase
 from internal.player import CustomPlayer
 from internal.load_later_track import LoadLaterTrack
-from config import spotify_client_id, spotify_client_secret
-
+from bot_base import BotBaseBot
```

Comment:
bot_base file is outside the cogs folder, pls confirm the location here.

Reply:
yup it's fine